src/momento/responses/list_data/list_length.py

```diff
@@ -0,0 +1,50 @@
+from abc import ABC
+from dataclasses import dataclass
+
+from momento.errors import SdkException
+
+from ..mixins import ErrorResponseMixin
+from ..response import CacheResponse
+
+
+class CacheListLengthResponse(CacheResponse):
+    """Response type for a `list_length` request. Its subtypes are:
+
+    - `CacheListLength.Hit`
+    - `CacheListLength.Miss`
+    - `CacheListLength.Error`
+
+    See `SimpleCacheClient` for how to work with responses.
+    """
+
+
+class CacheListLength(ABC):
+    """Groups all `CacheListLengthResponse` derived types under a common namespace."""
+
+    @dataclass
+    class Hit(CacheListLengthResponse):
+        """Indicates the list exists and its length was fetched."""
+
+        length: int
+        """The number of values in the list.
+
+        Returns:
+            int
```

Comment:
No need to document this as a return value; it's an attribute of the class. The first line should do